### ISSUE TITLE
reject X on selected thumbs not red

### ISSUE THREAD

**Issue by TurboGit** [darktable-org/darktable#4795]
**Describe the bug**
reject X on selected thumbs not red

**To Reproduce**
1. Go to lighttable
2. Click on X of one image
3. Let this thumbs selected and move over another thumb
4. See, the X is not read anymore

**Expected behavior**

The reject flag X should be red even when image selected.

**Screenshots**

**Platform (please complete the following information):**
 - Darktable Version: master
 - OS: Debian

**Additional context**

**Comment by TurboGit** [comment]
@Nilvus : Another one for you I fear. Nothing urgent.

**Comment by Nilvus** [comment]
Indeed, just missing selected tag... Will see that when testing preferences rework until this week-end.

Could you assign this and the #4794 issue also to me as a reminder? I will post one PR for both these and preferences CSS improvement from @elstoc PR at the same time.

**Comment by TurboGit** [comment]
@Nilvus : done.

**Comment by Nilvus** [comment]
@TurboGit: this is fix on my side. As I will not post my PR now (I will now work on preferences rework and I'm waiting also to a new to change on this part), here the tweak for that:

```
/* reject */
#thumb_main:hover #thumb_reject:active,
.dt_overlays_always #thumb_reject:active,
**.dt_overlays_always #thumb_main:selected #thumb_reject:active,**
.dt_overlays_always_extended #thumb_reject:active,
**.dt_overlays_always_extended #thumb_main:selected #thumb_reject:active,**
.dt_overlays_mixed #thumb_reject:active,
**.dt_overlays_mixed #thumb_main:selected #thumb_reject:active**
{
  color: red;
}
```

Just add the three selected lines above on same part beginning at line 1632 of darktable.css

Is it ok on your side with that change?

**Comment by TurboGit** [comment]
Works for me... Do you want me to commit that ?

**Comment by Nilvus** [comment]
> Works for me... Do you want me to commit that ?

If you want, that will be faster. So if you are on that, could you also add the CSS fix I post on this issue: #4794 (a tiny fix at then of darktable.css file) and so close these two issues.

That would be faster, I think CSS preferences rework will be a bit longer, some things are discussed by now to improve some Gtk parts before tweaking CSS

**Comment by TurboGit** [comment]
All committed now. Thanks.
